alfresco-docker-transform-misc/LICENSES.md

```diff
@@ -0,0 +1,11 @@
+### Licenses
+
```

Comment:
If you copied this from the other T-Engines, please raise an ATS jira to say that this should be changed in all 5 standard T-Engines as they will be used in community now. If this has changed in the other T-Engines, please make the text the same. I know the InT team were changing licenses a while back.